packages/render-fidelity-tools/test/results-viewer.html

```diff
@@ -103,24 +103,72 @@ <h2>Render Fidelity Comparison Results</h2>
     as real-time rendering at this level of quality is still very much an area
     of active research with improvements being made constantly. This site
     demonstrates where we are on that path to convergence and highlights areas
-    that could still use improvement. We are comparing the most popular web
-    renderers: <a href="https://threejs.org/" target="_blank" rel="noopener">three.js</a>
-    (represented by &lt;model-viewer&gt;), <a
-    href="https://google.github.io/filament/webgl/"
-    target="_blank" rel="noopener">filament.js</a>, <a href="https://www.babylonjs.com/"
-    target="_blank" rel="noopener">babylon.js</a>, <a href="https://github.khronos.org/glTF-Sample-Viewer-Release/"
-    target="_blank" rel="noopener">gltf-sample-viewer</a>, and <a href="https://github.com/gkjohnson/three-gpu-pathtracer/"
-    target="_blank" rel="noopener">three-gpu-pathtracer</a>. If any other renderers would like to be
+    that could still use improvement. We are comparing the most popular real-time web
+    renderers as well as path tracers (a rendering technique that uses far fewer approximations than are required by real-time renderers):</p>
+    <table class="browser-support">
+      <tr>
+        <th>Name</th>
+        <th>Rendering Type</th>
+        <th>Platform</th>
+     <tr>
+        <td><a href="https://threejs.org/" target="_blank" rel="noopener">three.js</a>
+          (represented by <a
+          href="https://modelviewer.dev"
+          target="_blank" rel="noopener">&lt;model-viewer&gt;)</a></td>
+        <td>Real-Time</td>
+        <td>Web</td>
+        </tr>
+            <tr>
+            <td><a
+          href="https://google.github.io/filament/webgl/"
+          target="_blank" rel="noopener">filament.js</a></td>
+          <td>Real-Time</td>
+          <td>Web, Android</td>
+          </tr>
+        <tr>
+            <td><a href="https://www.babylonjs.com/"
+          target="_blank" rel="noopener">babylon.js</a></td>
+          <td>Real-Time</td>
+          <td>Web</td>
+      </tr>
+        <tr>
+            <td><a href="https://github.khronos.org/glTF-Sample-Viewer-Release/"
+          target="_blank" rel="noopener">gltf-sample-viewer</a></td>
+          <td>Real-Time</td>
+          <td>Web</td>
+    </tr> 
+    <tr>
+      <td><a
+        href="https://www.3ds.com/products-services/3dexcite/resource-center/stellar-physically-correct/"
+        target="_blank" rel="noopener">Dassault STELLAR</a></td>
+    <td>Path Tracing</td>
+    <td>Unknown</td>
```

Comment:
@bsdorra Care to tell us about the platforms STELLAR runs on?

Reply:
Linux and Windows 

More specifically:
GPU (x86_64 host only): Maxwell, Pascal, Volta, Turing, Ampere, Ada
CPU: 
AVX2, AVX512 
SSE2, SSE41, AVX (legacy render modes only)
ARMv8.2-A

Reply:
I've updated the page to include the Stellar platform info!